### PR TITLE
doc: added channel at slack and added new qq group.

### PR DESCRIPTION
**Pull request by moonming**
### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible?

### CODE REVIEW

**Comment by Miss-you** [comment]
image
836398818

File: README.md [hunk 1]
```diff
@@ -23,9 +23,11 @@
 [![Build Status](https://travis-ci.org/apache/apisix.svg?branch=master)](https://travis-ci.org/apache/apisix)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/apache/apisix/blob/master/LICENSE)
 
-- **QQ group**: 552030619
 - Mail list: Mail to dev-subscribe@apisix.apache.org, follow the reply to subscribe the mail list.
+- **QQ group**: 578997126(recommend), 552030619
```
Comment (juzhiyuan) [inline README.md:27]: QQ Group would be better.

Also please use recommended

File: README.md [hunk 2]
```diff
@@ -23,9 +23,11 @@
...
+- Join `apisix` channel at [Apache Slack](http://s.apache.org/slack-invite). If the link is not working, find the latest one at [Apache INFRA WIKI](https://cwiki.apache.org/confluence/display/INFRA/Slack+Guest+Invites).
```
Comment (juzhiyuan) [inline README.md:28]: Join in

**Review by juzhiyuan — APPROVED** [review]
LGTM anyway

**Review by juzhiyuan — APPROVED** [review]

File: README_CN.md [hunk 3]
```diff
@@ -23,9 +23,11 @@
 [![Build Status](https://travis-ci.org/apache/apisix.svg?branch=master)](https://travis-ci.org/apache/apisix)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/apache/apisix/blob/master/LICENSE)
 
-- **QQ 交流群**: 552030619
 - 邮件列表: 发邮件到 dev-subscribe@apisix.apache.org, 然后跟着回复邮件操作即可。
+- **QQ 交流群**: 578997126(推荐), 552030619
```
Comment (Miss-you) [inline README_CN.md:27]: LGTM

**Review by Miss-you — APPROVED** [review]
ok

**Pull request merged**